empress/recon_vis/recon_viewer.py

```diff
@@ -36,9 +44,7 @@ def render(host_dict: dict, parasite_dict: dict, recon_dict: dict, event_frequen
     if show_legend:
         _create_legend(fig, consistency_type)
 
-    # Calculates font sizes
-    num_tip_nodes = len(host_tree.leaf_list()) + len(parasite_tree.leaf_list())
-    font_size = _calculate_font_size(num_tip_nodes)
+    font_size = node_font_size
```

Comment:
Instead of calculating the font size using the sigmoid function, use the default font size of 0.3 and add a font size function parameter.